docs/tutorial/part05.rst

```diff
@@ -2,5 +2,118 @@
 How to create the API.
 ======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+Why the API structure
+=====================
+
+We decided to use APIs to fetch the data to make queries more effective, as sometimes queries get too big, we decided it was better to fetch the data in little chunks rather than one big request and that's what APIs allows us to do.
+
+Explain the library
+===================
+
+In this project, we are using the rest_framework_ library because it has some simple non-limiting ways to implement an API structure, if you desire you can use any of your choosing, but the examples presented ahead will be based on the behavior of this specific library.
+
+    .. _rest_framework: http://www.django-rest-framework.org/
+
+Install the library
+===================
+
+Just run::
+
+    pip install rest_framework

```

Comment:
This is already in the requirement file; maybe the user already has this if he is following the tutorial. Maybe a note pointing this should be useful?
What do you think? 🤔